src/datasources/cache/cache.first.data.source.ts

```diff
@@ -44,10 +44,10 @@ export class CacheFirstDataSource {
   ): Promise<T> {
     const cached = await this.cacheService.get(cacheDir);
     if (cached != null) {
-      winston.debug(`[Cache] Cache hit: ${cacheDir.key}`);
+      winston.debug(`[Cache] Cache hit: [${cacheDir.key}, ${cacheDir.field}]`);
```

Comment:
Just adding the field key part to the logs output for better visibility. We are using the field to store addresses in this PR so I think it makes sense to include that in the logs now.